gremlin/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: gremlin
-version: 0.4.8
+version: 0.5.0-alpha
 description: The Gremlin Inc client application
 appVersion: "2.16.2"
```

Actually, this is a good idea too, indicating `2.25.2` will be the minimum required version here.
```suggestion
appVersion: "2.25.2"
```